notebooks/agent-breakpoints.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>Agentic - agentic (adjective in lowercase)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>